vendor.conf

```diff
@@ -3,10 +3,11 @@ github.com/asaskevich/govalidator f9ffefc3facfbe0caee3fea233cbb6e8208f4541
 github.com/Azure/go-ansiterm d6e3b3328b783f23731bc4d058875b0371ff8109
 github.com/beorn7/perks 3a771d992973f24aa725d07868b467d1ddfceafb
 github.com/containerd/console c12b1e7919c14469339a5d38f2f8ed9b64a9de23
-github.com/containerd/containerd d97a907f7f781c0ab8340877d8e6b53cc7f1c2f6
+github.com/containerd/containerd 9b32062dc1f5a7c2564315c269b5059754f12b9d # v1.2.1
 github.com/containerd/continuity f44b615e492bdfb371aae2f76ec694d9da1db537
-github.com/containerd/fifo 3d5202a
-github.com/containerd/typeurl f694355
+github.com/containerd/cri 0ca1e3c2b73b5c38e72f29bb76338d0078b23d6c # release/1.2 branch
```

Comment:
We can get rid of this dependency once https://github.com/docker/distribution/pull/2786 is accepted (and containerd/containerd and containerd/cri are updated accordingly)

Reply:
Are containerd/containerd and containerd/cri updated? Is this comment still valid?

Reply:
The PR was merged in docker/distribution, but it'll take some time before it to get into containerd/cri and a containerd release

Reply:
Ok so I guess in a followup then 👍